runGatling.sh

```diff
@@ -154,6 +155,7 @@ runNativeBinaryTests(){
   kill -9 "$(lsof -t -i :8080)" || true
 }
 
+test "avaje-jex-jdk/target/avaje-jex-jdk-$AVAJE.jar" "Avaje Jex started class sun.net.httpserver.HttpServerImpl"
```

Comment:
```suggestion
test "avaje-jex-jdk/target/avaje-jex-jdk-$AVAJE.jar" "Avaje Jex started class sun.net.httpserver.HttpServerImpl" "AVAJE" "https://avaje.io/"
```
we use that web link on test result to link to project page. the capital "AVAJE" can be used title link too,  spring example
<img width="737" height="76" alt="image" src="https://github.com/user-attachments/assets/4e100106-8565-48d8-a35e-84fc8b5c4476" />
I check your starting logs I dont see took XX seconds like entry, it would help people to see how long it take to start like in others, like in micronaut
<img width="805" height="138" alt="image" src="https://github.com/user-attachments/assets/3a54b4d1-444f-455b-83f4-5dc7befea589" />